x-pack/legacy/plugins/siem/public/containers/case/api.ts

```diff
@@ -45,70 +59,74 @@ export const getCases = async ({
     sortOrder: 'desc',
   },
 }: FetchCasesProps): Promise<AllCases> => {
-  const stateFilter = `case-workflow.attributes.state: ${filterOptions.state}`;
+  const stateFilter = `${CaseSavedObjectType}.attributes.state: ${filterOptions.state}`;
   const tags = [
-    ...(filterOptions.tags?.reduce((acc, t) => [...acc, `case-workflow.attributes.tags: ${t}`], [
-      stateFilter,
-    ]) ?? [stateFilter]),
+    ...(filterOptions.tags?.reduce(
+      (acc, t) => [...acc, `${CaseSavedObjectType}.attributes.tags: ${t}`],
+      [stateFilter]
+    ) ?? [stateFilter]),
   ];
   const query = {
     ...queryParams,
-    filter: tags.join(' AND '),
-    search: filterOptions.search,
+    ...(tags.length > 0 ? { filter: tags.join(' AND ') } : {}),
+    ...(filterOptions.search.length > 0 ? { search: filterOptions.search } : {}),
   };
-  const response = await KibanaServices.get().http.fetch(`${CASES_URL}`, {
+  const response = await KibanaServices.get().http.fetch<CasesResponse>(`${CASES_URL}/_find`, {
```

Comment:
Why the _find??

Reply:
that's the way we did it for rules and it is also to match what saved object do behind the scene.